docs/en/writing-running-appium/ios-xctest-mobile-gestures.md

```diff
@@ -0,0 +1,182 @@
+## Automating Mobile Gestures For iOS with WDA/XCTest backend
+
+Unfortunately Apple's XCTest framework does not natively support W3C standards for
+TouchAction interface implementation. Although, it provides rich set of gestures,
+inluding these, that are unique for iOS platform. It is possible to directly invoke these
+gestures in Appium starting from version 1.6.4-beta.
+
+It is important to rememeber that XCTest and WDA are being constantly changed.
+This means all "mobile: *" commands can be also subject of change in Appium
+without any preliminary notice.
+
+
+### mobile: swipe
+
+This gesture performs a simple "swipe" gesture on the particular screen element or
+on the application element, which is usually the whole screen.
+
+#### Supported arguments
+
+ * _direction_: Either 'up', 'down', 'left' or 'right'. The parameter is mandatory
+ * _element_: Element id to swipe on. Application element will be used instead if this
+ parameter is not provided
+
+#### Usage examples
+
+```java
+// Java
+JavascriptExecutor js = (JavascriptExecutor) driver;
+HashMap<String, String> scrollObject = new HashMap<String, String>();
+scrollObject.put("direction", "down");
+scrollObject.put("element", ((RemoteWebElement) element).getId());
+js.executeScript("mobile: swipe", scrollObject);
+```
+
+
+### mobile: scroll
+
+Scrolls the element or the whole screen. Different scrolling strategies are supported.
+Arguments define the choosen strategy: either 'name', 'direction', 'predicateString' or
+'toVisible' in that order. All strategies are exclusive and only one strategy
+can be applied at a single moment of time.
+
```

Comment:
It'll be nice to mention when to use `scroll` and when to use `swipe`.  From issues and forum, I found this is always confusing for new comers of appium. Please review and add some details